src/DynamoCoreWpf/ViewModels/Watch3D/DynamoGeometryModel3D.cs

```diff
@@ -90,9 +90,17 @@ protected override RenderCore OnCreateRenderCore()
             return new DynamoGeometryMeshCore();
         }
 
+#if NET6_0_OR_GREATER
+        protected override IRenderTechnique OnCreateRenderTechnique(IEffectsManager effectsManager)
+        {
+            return effectsManager[DynamoEffectsManager.DynamoMeshShaderName];
+        }
+
+#elif NETFRAMEWORK
```

Comment:
so we need this change just because we have upgraded the helixToolkit pacakge version ?
Upgrading this pacakge does not guarantee it will run successfully on dotnet6

If the intention was to only build successfully (not figure out the dependencies), why did we upgrade the version ? Couldn't we have just kept the old versions and not have to modify the sources at all ?

Reply:
In order to just build successfully the project, I restored the old version. in new activities I will review the packages in detail.